201-vmss-lap-autoscale/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a VM Scale Set with Linux VMs and Auto Scale configured",
```

Comment:
Title and description need to be updated. Currently it is the same description as other vmss autoscale template.